### PR TITLE
[Enhancement] Add gender field and employee #

### LINKED ISSUE #146: [Enhancement] Add gender field and employee #
dependency of associatedemployers/granite#419

### PR DESCRIPTION
**Pull request by JavaLavaMT**
Closes #146

### CODE REVIEW

File: api/employee/parser-definition.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "name.middle":                 [ "middle name", "middle initial", "name middle", "mi", "m.i." ],
   "name.last":                   [ "lastname", "last name", "name last", "given name", "name given", "family name", "name family" ],
   "name.suffix":                 [ "suffix", "name suffix" ],
+  "gender":                      ["gender", "sex"],
```
[the remark this answers is not in the document]
Comment (JavaLavaMT) [reply api/employee/parser-definition.json]: 👍

File: lib/models/employee/index.js [hunk 2]
```diff
@@ -44,10 +44,17 @@ var employeeSchema = new Schema({
...
   },
 
-  picture:  String,
-  phone:    String,
-  ssn:      String,
-  password: String,
+  gender: {
+    type: String,
+    required: true,
+    enum: [ 'Male', 'Female', 'Other' ]
```
[the remark this answers is not in the document]
Comment (JavaLavaMT) [reply lib/models/employee/index.js]: 👍

**Comment by JavaLavaMT** [comment]
> A few changes. Please also fix Slate import logic to use these fields.

do you mean in slate census? or slate integration?

**Comment by James1x0** [comment]
@JavaLavaMT Disregard that comment, I made the change this week.

**Pull request merged**